include/z64actor.h

```diff
@@ -357,4 +357,13 @@ typedef enum {
     DOORLOCK_NORMAL_SPIRIT
 } DoorLockType;
 
+#define UPDBGCHECKINFOFLAG_0 (1 << 0) // check wall
+#define UPDBGCHECKINFOFLAG_1 (1 << 1) // check ceiling
+#define UPDBGCHECKINFOFLAG_2 (1 << 2) // check floor and water
+#define UPDBGCHECKINFOFLAG_3 (1 << 3)
+#define UPDBGCHECKINFOFLAG_4 (1 << 4)
+#define UPDBGCHECKINFOFLAG_5 (1 << 5) // unused
+#define UPDBGCHECKINFOFLAG_6 (1 << 6) // disable water ripples
+#define UPDBGCHECKINFOFLAG_7 (1 << 7) // alternate wall check?
```

Comment:
```suggestion
#define UPDBGCHECKINFOFLAG_0 (1 << 0) // check wall
#define UPDBGCHECKINFOFLAG_1 (1 << 1) // check ceiling
#define UPDBGCHECKINFOFLAG_2 (1 << 2) // check floor and water
#define UPDBGCHECKINFOFLAG_3 (1 << 3)
#define UPDBGCHECKINFOFLAG_4 (1 << 4)
#define UPDBGCHECKINFOFLAG_5 (1 << 5) // unused
#define UPDBGCHECKINFOFLAG_6 (1 << 6) // disable water ripples
#define UPD_BGCHECK_INFO_FLAG_7 (1 << 7) // alternate wall check?
```
I know it's getting a bit verbose, but shouldn't these be spaced out by `_`? Seems hard to read as one big blog of text
And also should `BGCHECK_` take a higher priority than `UPD`? So would `BGCHECK_UPD_INFO_FLAG_7` make more sense?

Reply:
the scope of these flags is really the one function: `Actor_UpdateBgCheckInfo`
a bgcheck prefix makes it seem like its a part of bgcheck imo, and the whole "unit" or "system" is "update bgcheck info"

the name is not ideal at all as dragorn mentioned, and hopefully we can rename these variables all together to be less verbose

as far as spacing them goes, `UPDBGCHECKINFO_FLAG_7` would prob work, at least for the time being